issue_event_processing.ipynb

```diff
@@ -0,0 +1,458 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction\n",
+    "Process mining helps uncover insights from data that represent different events in a process. It is valuable for understanding workflows, finding inefficiencies, and optimizing development processes. Using Kaiaulu  for process mining enables this discovery for Github Events (e.g. creating issues, push events, deleting branches, etc). Allowing users to visualize and analyze event data effectively. This notebook demonstrates how to use Kaiaulu to download and parse Github Event data. As well how to interact with the data to leverage process discovery for deeper analysis. This [video](https://www.youtube.com/watch?v=XLHtvt36g6U&t=1181s) provides an overview of process mining concepts and its applications and can be used as a reference point to begin. This notebook heavily relies on the python package [pm4py](https://github.com/process-intelligence-solutions/pm4py) to create event mappings.\n",
+    "\n",
+    "This notebook walks through multiple different models and algorithms for modeling the process. It is reccomended you start with a small number of event issues and gradually build multiple versions to have a deeper understanding of what events are changed the process model. At the bottom of the notebook is an examples section that shows what the models can generate. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Requirements\n",
+    "\n",
+    "-   Gihub Access Token \n",
+    "-   [Kaiaulu](https://github.com/sailuh/kaiaulu) R package to download data via CLI\n",
+    "-   Python environment with [pm4py](https://github.com/process-intelligence-solutions/pm4py) installed\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Python Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 137,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas\n",
+    "import os\n",
+    "from datetime import datetime\n",
+    "import pm4py\n",
+    "import subprocess\n",
+    "\n",
+    "from pm4py.algo.discovery.dfg import algorithm as dfg_discovery\n",
+    "from pm4py.algo.discovery.heuristics import algorithm as heuristics_miner"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download and Parse data with Kaiaulu ghevents.R (CLI)\n",
+    "\n",
+    "#### Set Up and Execute Download Command\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Save and set the current working directory to Kaiaulu\n",
+    "cwd = os.getcwd()\n",
+    "os.chdir(os.path.expanduser(\"~/Desktop/Kaiaulu/Working_issues/kaiaulu/\"))\n",
```

Comment:
Use relative paths, explicitly create a code block that says output and tell the user by default it will save to the same folder this code was downloaded (the only path you know for certain will exist on user side). 

You can also assume the Kaiaulu recommended path structure with raw_data folder exists.